cooler_manipulation.ipynb

```diff
@@ -0,0 +1,1524 @@
+{
```

Comment:
<pre class="ql-syntax" spellcheck="false">Line #2.    clr_new.bins()[:]
</pre><p>not sure I understand this explanation...  </p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/open2c/open2c_examples/pull/28/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>

Reply:
# Note after reordering, bin coordinates might not be multiples of the binsize. 
# This is because chromosome lengths are unlikely to be a multiple of binsize,
# and the last bin of a chromosome is often shorter.
# If a short bin that was formerly at the end of a chromosome is reordered to
# the start or middle of a chromosome, we observe bins positions that
# are not multiples of the binsize.